docs/reference/ccr/apis/auto-follow/pause-auto-follow-pattern.asciidoc

```diff
@@ -0,0 +1,85 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ccr-pause-auto-follow-pattern]]
+=== Pause auto-follow pattern API
+++++
+<titleabbrev>Pause auto-follow pattern</titleabbrev>
+++++
+
+Pauses an auto-follow pattern.
+
+[[ccr-pause-auto-follow-pattern-request]]
+==== {api-request-title}
+
+//////////////////////////
+
+[source,console]
+--------------------------------------------------
+PUT /_ccr/auto_follow/my_auto_follow_pattern
+{
+  "remote_cluster" : "remote_cluster",
+  "leader_index_patterns" :
+  [
+    "leader_index"
+  ],
+  "follow_index_pattern" : "{{leader_index}}-follower"
+}
+--------------------------------------------------
+// TEST[setup:remote_cluster]
+// TESTSETUP
+
+//////////////////////////
+
+[source,console]
```

Comment:
In this section, we typically use literals instead of an actual snippet:

`POST /_ccr/auto_follow/<auto_follow_pattern_name>/pause`

You may want to move the preceding test setup lower but no big deal.